config/initializers/assets.rb

```diff
@@ -7,7 +7,7 @@
 # Rails.application.config.assets.paths << Emoji.images_path
 # Add Yarn node_modules folder to the asset load path.
 Rails.application.config.assets.paths << Rails.root.join('node_modules')
-Rails.application.config.assets.precompile += %w( footer.js )
+# Rails.application.config.assets.precompile += %w( footer.js ) <---- this was removed - not sure if it should
```

Comment:
Question

Reply:
yes it is likely needed. its hard to tell in dev mode, but in prod mode you might see a 404 for the footer.js asset

Reply:
@aprilrieger rez back